### PR TITLE
Merge 4.5.1-7.10 into 4.6.0-7.10

### PR DESCRIPTION
**Pull request by Machi3mfl**
### Description

Merge 4.5.1-7.10 into 4.6.0-7.10

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
...
+- SysV Init:
+
+```
+service kibana restart
+```
+
+## Wazuh - Kibana - Open Distro version compatibility matrix
+
+The compatibility matrix is avaliable in the repository [wiki](https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility).
+
```
Comment (Desvelao) [inline README.md:244]: **nitpick**: this must be moved to `plugins/main/README.md`.

File: plugins/main/public/components/common/welcome/agents-info.js [hunk 2]
```diff
@@ -146,25 +154,64 @@ export class AgentInfo extends Component {
...
-        { title: agent.version, description: 'Version', style: { minWidth: 100 } },
+        {
+          title: agent.status,
+          description: 'Status',
+          style: { minWidth: 100 },
+        },
+        {
+          title: agent.ip,
+          description: 'IP address',
+          style: { minwidth: 150 },
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/agents-info.js:184]: **nitpick**: remove the `minwidth` property. It is unused ( and has a typo ).

File: plugins/main/public/controllers/agent/components/agents-preview.js [hunk 3]
```diff
@@ -234,94 +289,113 @@ export const AgentsPreview = compose(
...
+                              >
+                                <a
+                                  onClick={() =>
+                                    this.showAgent(
+                                      this.state.lastRegisteredAgent,
+                                    )
+                                  }
+                                >
+                                  {this.state.lastRegisteredAgent?.name || '-'}
+                                </a>
```
Comment (Desvelao) [inline plugins/main/public/controllers/agent/components/agents-preview.js:315]: **nipitck**: replace `a` tag with a `EuiLink` component.

File: plugins/main/public/controllers/agent/components/agents-preview.js [hunk 4]
```diff
@@ -234,94 +289,113 @@ export const AgentsPreview = compose(
...
-                                  <EuiLink onClick={() => this.showAgent(this.state.agentMostActive)}>
+                                <EuiToolTip
+                                  position='top'
+                                  content='View agent details'
+                                >
+                                  <a
+                                    onClick={() =>
+                                      this.showAgent(this.state.agentMostActive)
+                                    }
+                                  >
```
Comment (Desvelao) [inline plugins/main/public/controllers/agent/components/agents-preview.js:341]: **nitpick**: the opening `a` tag should be a `EuiLink` component.

File: plugins/main/public/controllers/agent/components/agents-table.js [hunk 5]
```diff
@@ -281,8 +310,16 @@ export const AgentsTable = withErrorBoundary(
     }
 
     formatAgent(agent) {
-      const agentVersion = agent.version !== undefined ? agent.version.split(' ')[1] : '-';
-      const node_name = agent.node_name && agent.node_name !== 'unknown' ? agent.node_name : '-';
+      const checkField = field => {
+        return field !== undefined ? field : '-';
+      };
```
Comment (Desvelao) [inline plugins/main/public/controllers/agent/components/agents-table.js:315]: **nitpick**: remove if this is unused.

File: plugins/main/public/controllers/agent/components/agents-table.js [hunk 6]
```diff
@@ -333,8 +384,11 @@ export const AgentsTable = withErrorBoundary(
     }
 
     addIconPlatformRender(agent) {
-      let icon = '';
-      const os = agent?.os || {};
+      let icon = false;
+      const checkField = field => {
+        return field !== undefined ? field : '-';
+      };
+      const os = (agent || {}).os;
```
Comment (Desvelao) [inline plugins/main/public/controllers/agent/components/agents-table.js:391]: **nitpick**: review the previous way to do it.

File: CHANGELOG.md [hunk 7]
```diff
@@ -53,14 +53,41 @@ All notable changes to the Wazuh app project will be documented in this file.
...
 
+## Wazuh v4.4.5 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 02
+
+### Added
+
+- Support for Wazuh 4.4.5
+
 ## Wazuh v4.4.4 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 01
 
+
```
Comment (Desvelao) [inline CHANGELOG.md:90]: **nitpick**: remove the new line.

File: plugins/main/scripts/tag.py [hunk 8]
```diff
@@ -17,7 +17,7 @@
 # Wazuh version: major.minor.patch
 version = '4.6.0'
 # App's revision number (previous rev + 1)
-revision = '01'
+revision = '02'
```
Comment (Desvelao) [inline plugins/main/scripts/tag.py:20]: **nipitck**: undone the change.

**Comment by Desvelao** [comment]
WIP
image
image

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 9]
```diff
@@ -59,495 +63,641 @@ import { webDocumentationLink } from '../../../../common/services/web_documentat
...
-            </div>
-          </EuiPopover>
-        </EuiFlexItem>
-      </Fragment>
-    )
-  }
-
-  renderTitle() {
-
-      const notNeedStatus = true;
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:263]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 10]
```diff
@@ -59,495 +63,641 @@ import { webDocumentationLink } from '../../../../common/services/web_documentat
...
+                      </WzReduxProvider>
+                    </div>
+                  </EuiPopover>
+                </EuiFlexItem>
+              )}
+              <EuiFlexItem className='wz-agent-empty-item'></EuiFlexItem>
               <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
                 <EuiButtonEmpty
-                  iconType="inspect"
-                  onClick={() => this.props.switchTab('syscollector', notNeedStatus)}>
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:326]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 11]
```diff
@@ -59,495 +63,641 @@ import { webDocumentationLink } from '../../../../common/services/web_documentat
...
+                  iconType='inspect'
+                  onClick={() => this.props.switchTab('syscollector')}
+                >
                   Inventory data
                 </EuiButtonEmpty>
               </EuiFlexItem>
               <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
                 <EuiButtonEmpty
-                  iconType="stats"
-                  onClick={() => this.props.switchTab('stats', notNeedStatus)}>
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:333]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 12]
```diff
@@ -59,495 +63,641 @@ import { webDocumentationLink } from '../../../../common/services/web_documentat
...
+                  iconType='stats'
+                  onClick={() => this.props.switchTab('stats')}
+                >
                   Stats
                 </EuiButtonEmpty>
               </EuiFlexItem>
               <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
                 <EuiButtonEmpty
-                  iconType="gear"
-                  onClick={() => this.props.switchTab('configuration', notNeedStatus)}>
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:340]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 13]
```diff
@@ -59,495 +63,641 @@ import { webDocumentationLink } from '../../../../common/services/web_documentat
...
+        return 'Wazuh-App-Agents-Welcome-Top-NIST-800-53';
+      }
+      if (this.state.selectedRequirement === 'gpg13') {
+        return 'Wazuh-App-Agents-Welcome-Top-GPG-13';
+      }
+      if (this.state.selectedRequirement === 'tsc') {
+        return 'Wazuh-App-Agents-Welcome-Top-TSC';
+      }
+      return 'Wazuh-App-Agents-Welcome-Top-PCI';
+    }
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: README.md (hunk 1 above)
Comment (Desvelao) [reply README.md]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: CHANGELOG.md (hunk 7 above)
Comment (Desvelao) [reply CHANGELOG.md]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

File: plugins/main/scripts/tag.py (hunk 8 above)
Comment (Desvelao) [reply plugins/main/scripts/tag.py]: Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f

**Review by Desvelao — DISMISSED** [review]

**Review by yenienserrano — APPROVED** [review]
LGTM

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by yenienserrano — APPROVED** [review]
LGTM!

**Pull request merged**